src/_guides/language/evolution.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Dart language evolution
+short-title: Language evolution
+description: Notable changes and additions to the Dart programming language.
+---
+
+This page lists notable changes and additions to the Dart programming language.
+If you want details about the currently supported language, see the
+[language tour][] or the
+[language specification][].
+
+For a peek into current features being discussed, investigated, and added to the
+Dart language, see the [language funnel][] tracker in the Dart
+language GitHub repo.
+
+## Dart 2.0
+
+Dart 2.0 implemented a new **[sound type system][]**. Before
+Dart 2.0, types weren't fully sound, and Dart relied heavily on runtime type
+checking. Dart 1.x code had to be [migrated to Dart 2][].
+
+## Dart 2.1
+
+Dart 2.1 added support for **int-to-double conversion**, allowing developers to
+set `double` values using integer literals. This feature removed the annoyance
+of being forced to use a `double` literal (for example, `4.0`)
+when the value was conceptually an integer.
+In the following Flutter code, `horizontal` and `vertical` have type `double`:
+
+```dart
+padding: const EdgeInsets.symmetric(
+  horizontal: 4,
+  vertical: 8,
+)
+```
+
+## Dart 2.2
+
+Dart has always supported literal lists and maps, but
+Dart 2.2 added support for **[set literals][]**:
+
+```dart
+const Set<String> currencies = {'EUR', 'USD', 'JPY'};
+```
+## Dart 2.3
+
+Dart 2.3 added three operators designed to improve code that performs
+list manipulation, such as declarative UI code.
```

Comment:
Not just lists - all primitive collections (lists, sets, maps).

Reply:
How about I change it to this:

...improve code (such as declarative UI code) that manipulates lists, sets, and maps.

Reply:
Works for me.

Reply:
Or maybe "lists, sets, and maps" -> "collections" or "primitive collections"